Fix rounding the Reimbursement total

mamhoff opened the pull request:
Previously, amounts that really should round up rounded down:
19.99999 would round to 19.99. This is unfortunate. Since every return item
should individually have a price that rounds to two digits, and rounding
individually (I think) solves the parent commit's issue, I propose
just rounding individually.

mamhoff committed:
Fix rounding the Reimbursement total

Previously, amounts that really should round up rounded down:
19.99999 would round to 19.99. This is unfortunate. Since every return item
should individually have a price that rounds to two digits, and rounding
individually (I think) solves the parent commit's issue, I propose
just rounding individually.

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://magiclabs-fix-reimbursement-rounding.spree-spree-9ed724b77adec04c49c2.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://magiclabs-fix-reimbursement-rounding.spree-spree-9ed724b77adec04c49c2.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

tvdeyen commented:
:+1: Thanks for all your afford into spree lately!

moritzps commented:
:+1:

mamhoff committed:
Remove all instances of rounding down

In financial calculations, please please please don't just randomly
round down. I found two more instances of random roundings, both of which
could quite easily be cured by rounding individual return items before
they are summed up.
